webapp/src/state/withLayout.js

```diff
@@ -59,7 +62,10 @@ export default function withLayout(options) {
         <div className={clsx(bg, "root", noBottomCls)}>
           {scrollTop && <ScrollTopOnMount />}
           <div className="main-container">
-            {nav === "top" && <TopNav />}
+            <div className="sticky-top">
```

Comment:
We don't want this div if there's no nav: `const hasNav = nav || appNav` and then `{hasNav && <div...}`